cartridges/LINK_adyen/cartridge/templates/default/checkout/billing/paymentOptions.isml

```diff
@@ -1,13 +1,9 @@
 <isscript>
     var assets = require('*/cartridge/scripts/assets.js');
     assets.addJs('/js/adyen-cse.js');
-
+    assets.addCss('/css/adyenCss.css');
```

Comment:
should this be /scss/adyenCss.scss ?

Reply:
@maassenbas can you reply on this comment of @Aleffio  ?

Reply:
No, this is the right path. After compiling, this code will occur in /scss/adyenCss.scss.